src/Listener/SentryExceptionListener.php

```diff
@@ -0,0 +1,67 @@
+<?php declare(strict_types=1);
+
+namespace Becklyn\Hosting\Listener;
+
+use Sentry\SentryBundle\EventListener\ExceptionListener;
+use Sentry\SentryBundle\EventListener\SentryExceptionListenerInterface;
+use Symfony\Component\Console\Event\ConsoleCommandEvent;
+use Symfony\Component\Console\Event\ConsoleErrorEvent;
+use Symfony\Component\Console\Event\ConsoleExceptionEvent;
+use Symfony\Component\HttpKernel\Event\GetResponseEvent;
+use Symfony\Component\HttpKernel\Event\GetResponseForExceptionEvent;
+
+final class SentryExceptionListener implements SentryExceptionListenerInterface
+{
+    /**
+     * @var ExceptionListener
+     */
+    private $inner;
+
+
+    public function __construct (ExceptionListener $inner)
+    {
+        $this->inner = $inner;
+    }
+
+
+    /**
+     * @inheritDoc
+     */
+    public function onKernelRequest (GetResponseEvent $event) : void
+    {
+
```

Comment:
Can we add a comment here on why this method is empty?

Reply:
Done.